config/htcondor-ce.spec

```diff
@@ -477,6 +480,7 @@ fi
 %config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-ce-collector.conf
 %config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-ce-collector-requirements.conf
 %config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-ce-auth.conf
+%config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-ce-collector-audit-payload.conf
```

Comment:
I wasn't sure.  What is the htcondor-ce-collector rpm for?  It already had other files that were in the main htcondor-ce so I assumed it was an rpm if somebody wanted to alternately install only the collector.